@xen-orchestra/web-core/lib/components/ui/panel/UiPanel.vue

```diff
@@ -0,0 +1,57 @@
+<!-- wip -->
+<template>
+  <div class="ui-panel" :class="{ error: props.error }">
```

Comment:
To keep consistency with the rest of the project, you can remove the `props.` and use the shorthand version:
```suggestion
  <div class="ui-panel" :class="{ error }">
```
And in the script tag:
```ts
defineProps<{
  error?: boolean
}>()
```